Symfony/CS/Test/AccessibleObject.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Test;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
```

Comment:
Personally I have no need for that, it's already final and functionality is so extra small that interface won't change.